Gui/python/TestHandler.py

```diff
@@ -368,10 +370,62 @@ def runSingleTest(self, testName):
             self.IVCurveResult = ScanCanvas(
                 self, xlabel="Voltage (V)", ylabel="I (A)", invert=True
             )
-            self.IVCurveHandler = IVCurveHandler(self, self.HVpowersupply)
-            self.IVCurveHandler.finished.connect(self.IVCurveFinished)
-            self.IVCurveHandler.IVCurve()
-            return
+
+            channel = 1
+            # copy from QtStartWindow
+            # find the reading values
+            LVStatusValue = self.LVpowersupply.Instrument.status(
+                channel, no_lock=True
+            )  # return 1 if it is on, 0 if it is off
+            if LVStatusValue:
```

Comment:
This doesn't do anything, in python 1 and 0 evaluate to true and false automatically.